packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -570,7 +588,7 @@ export class DocumentsSchemaController {
 	 * Please consider note above constructor about race conditions - current design is to send op only once in a session lifetime.
 	 * @returns Optional message to send.
 	 */
-	public maybeSendSchemaMessage(): IDocumentSchemaChangeMessage | undefined {
+	public maybeSendSchemaMessage(): IDocumentSchemaChangeMessageOutgoing | undefined {
 		if (this.sendOp && this.futureSchema !== undefined) {
 			this.sendOp = false;
 			assert(
```

Comment:
If you agree, I think you can remove `refSeq: this.documentSchema.refSeq` in the return statement.  No need to rock the boat if you don't want to though